compressor/compressor.h

```diff
@@ -0,0 +1,13 @@
+#include <cstdint>
+
+const int window_width = 512;//Won't work if more
```

Comment:
Better enclose all this consts in:
namespace Cfg {
}
and use with Cfg:: prefix.
Cause it can be Config as well.